### DISCUSSION TITLE
How to disable animations in tab strip?

### DISCUSSION THREAD

**Discussion by opzch** [black7375/Firefox-UI-Fix#837] [Q&A]
Would it be possible to disable all kinds of animations in the tab strip, or make it follow system preference? Currently, if you have opened several tabs enough to change the tab width, close the ones to the left with a mouse click and move the cursor, the rest "smoothly" expands and slides right (regardless of "disable animations" preference in browser settings or system settings). For some reason, if you use a keyboard shortcut to close them, they appear in their places instantly, like they should.

https://github.com/black7375/Firefox-UI-Fix/assets/99139463/29819c48-05fd-4c54-922a-1f95051e45aa

**Comment by black7375** [comment]
Well, will this setup work?
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

**Reply by opzch** [reply]
Unfortunately, it will not, that's why I specified that tabs animate regardless of "disable animations" preference in browser settings (about:config > ui.prefersReducedMotion) or system settings (Windows 10: Settings > Ease of Access > Display > Show animations in Windows). Turning off both and separately enabling/disabling them does not affect this tab sliding animation.

Moreover, this seems to be the default behavior (as been noted https://bugzilla.mozilla.org/show_bug.cgi?id=1826839#c2 - "disable animations" is schizophrenically not "disable animations" because it is "great for usability"), so I'm trying to figure out some way to override it.
